Sprint_4

mshulgina opened the pull request:
проект

mshulgina committed:
Sprint_4

проект

alstafeev commented on src/main/java/pageObjects/MainPage.java:
```diff
@@ -0,0 +1,75 @@
+    //Локатор для кнопки Заказать внизу экрана
+    private static final By buttonOrderDown= By.className("Button_Button__ra12g Button_Middle__1CSJM");
+    // Локаторы кнопок с вопросами
+    public static final String[] accordionItem = new String[]{
```
Надо исправить - локаотры надо сделать private

alstafeev commented on src/main/java/pageObjects/MainPage.java:
```diff
@@ -0,0 +1,75 @@
+            "accordion__heading-6",
+            "accordion__heading-7"};
+    //Локаторы текстов "Вопросы о важном
+    public static final String[]  openPanel = new String[]{
```
Надо исправить - локаотры надо сделать private

alstafeev commented on src/main/java/pageObjects/OrderPage.java:
```diff
@@ -0,0 +1,104 @@
+        new WebDriverWait(driver, 10).until(ExpectedConditions.
+                elementToBeClickable(driver.findElement(By.xpath(".//button[text()='Да']"))));
+        driver.findElement(buttonYes).click();
+        TimeUnit.SECONDS.sleep(5);
```
Надо исправить - удалить sleep. Заменить на WebDriverWait

alstafeev commented on src/test/java/DropDownListTests.java:
```diff
@@ -0,0 +1,81 @@
+        }
+
+        @Test
+        public void test() throws InterruptedException {
```
Надо исправить - переименовать метод - дать понятное название что проверяет

alstafeev commented on src/test/java/DropDownListTests.java:
```diff
@@ -0,0 +1,81 @@
+            assertEquals("Иной текст", answerText, actualAnswerText);
+        }
+
+        @After
```
Можно лучше - вынести в базовый тестовый класс

alstafeev commented on src/test/java/DropDownListTests.java:
```diff
@@ -0,0 +1,81 @@
+            };
+        }
+
+        @Before
```
Можно лучше - вынести в базовый тестовый класс

alstafeev commented on src/test/java/OrderTests.java:
```diff
@@ -0,0 +1,105 @@
+            mainPage.clickButtonOrderDown();
+            OrderPage formForOrder = new OrderPage(driver);
+            //заполняем форму заказа
+            formForOrder.filling(name, surname, address, metroStation,
```
Надо исправить - нужно добавить проверку. Тест должен всегда что то проверять

alstafeev commented on src/test/java/OrderTests.java:
```diff
@@ -0,0 +1,105 @@
+                    {"Николай", "Ковальчук", "Воронеж", "Чистые пруды", "89999245432", "Описание заказа 1"},
+            };
+        }
+    @Before
```
Можно лучше - вынести в базовый тестовый класс

alstafeev commented on src/test/java/OrderTests.java:
```diff
@@ -0,0 +1,105 @@
+            formForOrder.filling(name, surname, address, metroStation,
+                    phone, commentField);
+        }
+    @After
```
Можно лучше - вынести в базовый тестовый класс

mshulgina committed:
исправленный проект

mshulgina committed:
исправлены ошибки

mshulgina committed:
исправлены опечатки после копирования id

mshulgina committed:
Локаторы на месте! PageObject в src/main!

Локаторы были исправлены еще в предыдущем пуше! "Так же надо перенести PageObject обратно в src/main" Куда обратно?! Они здесь были! я их не трогала. Пакет в SRC/MAIN. Я ВСЕ ИСПРАВИЛА. НЕ ВАЛИТЕ МОЮ РАБОТУ ПОЖАЛУЙСТА!!!!
